.travis.yml

```diff
@@ -0,0 +1,51 @@
+os: osx
+osx_image: xcode9.3beta
+
+env:
+  global:
+    - BUILD_CCACHE_DIR=~/build/ccache
+    - BUILD_NINJA_DIR=~/build/ninja
+  matrix:
+    - PYTHON_VERSION=python2
+    - PYTHON_VERSION=python3
+
+before_install:
+  - ./travis/before_install.sh
+
+install:
+  - ./travis/install.sh
+
+script:
+  - ./travis/script.sh
+
+after_success:
+  - ./travis/after_success.sh
+
+after_failure:
+  - ./travis/after_failure.sh
+
+cache:
+  - timeout: 300
+  - directories:
+    - $BUILD_CCACHE_DIR
+    - $BUILD_NINJA_DIR
+    - $HOME/.ccache
+    - $HOME/.cache/pb
+
+# We don't want to trigger travis builds for auto PR because there are
+# too many of them and they are updated very frequently. travis can
+# barely catchup and so causing the build status of these PRs always
+# being shown as pending.
+
+# For each PR, travis triggers two builds, one is "branch updates"
+# which uses the commit in the PR branch, another one is "pull request
+# updates" which uses the merge commit that merges the PR branch into
+# master branch. The following branches filter will only be able to
+# disable builds of "branch updates", but not the "pull request
+# updates", because the latter has branch name "master". To achieve
+# that, we have turned off "Build pull request updates" option in the
+# travis website.
+
+branches:
+  except:
+    - /^auto-.*$/
```

Comment:
You should remove this because travis is all you have right now :-) (unlike onnx-fb-universe which mainly uses the jenkins CI)

Reply:
Fixed